src/argilla/client/feedback/schemas/questions.py

```diff
@@ -122,6 +122,24 @@ class RatingQuestion(QuestionSchema, LabelMappingMixin):
     type: Literal["rating"] = Field("rating", allow_mutation=False)
     values: List[int] = Field(unique_items=True, min_items=2)
 
+    @validator("values")
+    def check_values(cls, values: List[int]):
+        if len(values) > 10:
+            warnings.warn(
+                "Values list contains more than 10 elements, which is not supported for newer versions of Argilla. "
+                "Please, review this settings before pushing the dataset and records into Argilla. "
+                "Otherwise, the `push_to_argilla action may fail",
+            )
+        for value in values:
+            if not 1 <= value <= 10:
+                warnings.warn(
+                    "Value found out of range [1, 10], which is not supported for newer versions of Argilla. "
+                    "Please, review this settings before pushing the dataset and records into Argilla. "
+                    "Otherwise, the `push_to_argilla action may fail. ",
```

Comment:
```suggestion
                    "At least one `value` in `values` is out of range [1, 10], which is not supported from Argilla 1.14.0 onwards. "
                    "Please, make sure `values` is a list with unique values within the range [1, 10] before pushing the dataset into Argilla. "
                    "Otherwise, the `push_to_argilla` method will fail. ",
```